js/samples/04.ai.a.teamsChefBot/.vscode/launch.json

```
@@ -90,6 +90,18 @@
                 "order": 2
             },
             "stopAll": true
+        },
+        {
+            "name": "Debug Local",
```

I think we want to indicate that this is for the emulator
```suggestion
            "name": "Debug Local in BF Emulator",
```